alacritty/src/config/cursor.rs

```diff
@@ -0,0 +1,149 @@
+use std::cmp;
+
+use serde::Deserialize;
+
+use alacritty_config_derive::{ConfigDeserialize, SerdeReplace};
+use alacritty_terminal::ansi::{CursorShape as VteCursorShape, CursorStyle as VteCursorStyle};
+
+use crate::config::ui_config::Percentage;
+
+const MIN_BLINK_INTERVAL: u64 = 10;
```

Comment:
Comment that tells me this is in `ms` would be nice.

"Minimum cursor blinking interval in milliseconds." or something.